services/navigation/builtin/builtin.go

```diff
@@ -196,14 +200,31 @@ func (svc *builtIn) Reconfigure(ctx context.Context, deps resource.Dependencies,
 		}
 		svc.logger = logger
 	}
+
 	base1, err := base.FromDependencies(deps, svcConfig.BaseName)
 	if err != nil {
 		return err
 	}
-	movementSensor, err := movementsensor.FromDependencies(deps, svcConfig.MovementSensorName)
-	if err != nil {
-		return err
+
+	var mapType navigation.MapType
```

Comment:
[note] Do we want this check to occur in validate? My initial reason for including it here is that Validate seems primarily used for the presences of config vars/setting defaults.